contracts/0.8.25/vaults/StakingVault.sol

```diff
@@ -315,9 +315,7 @@ contract StakingVault is IStakingVault, OwnableUpgradeable {
      * @dev Can only be called by VaultHub; locked amount can only be increased
      * @param _locked New amount to lock
      */
-    function lock(uint256 _locked) external {
-        if (msg.sender != address(VAULT_HUB)) revert NotAuthorized("lock", msg.sender);
-
+    function lock(uint256 _locked) external onlyOwner {
```

Comment:
In this configuration, it's possible to enforce _locked <= valuation, but it means that we should prefund 1 ETH in vault factory for deposit, which is fine tbh.